Android/app/src/main/java/com/example/brewhaha_android/Controllers/HomeActivity.kt

```diff
@@ -69,9 +100,69 @@ class HomeActivity(private val api: BackendConnection = BackendConnection()) : A
         }
     }
 
-    fun viewBreweries(tokenBundle: Bundle) {
-        val intent = Intent(baseContext, ViewBreweryActivity::class.java)
-        intent.putExtra("bundle", tokenBundle)
-        startActivity(intent)
+    fun getBreweries() {
+        val token = AuthToken(tokenBundle!!["token"] as String, "", "")
+        Log.d("Home Brewery Call", "getting all breweries")
+        doAsync {
+            val response = api.getAllBreweries(token).execute()
+            if (response.isSuccessful) {
+                breweryList = response.body()
+                Log.d("Home Brewery Call", breweryList.toString())
+                uiThread {
+                    Log.d("Home Brewery Call", "success")
+                    Log.d("Home Brewery Call", "Size: " + breweryList.size)
+                    breweryList.forEach {
+                        Log.d("Home Brewery Call", "Name: " + it)
+                    }
```

Comment:
Over here set the list view once you get a succesful response.

Reply:
I do set the list when I get a successful response. 4 lines above this

Reply:
Set the list in the RecyclerView